### PR TITLE
attempt cursed lighting

### PR DESCRIPTION
**Pull request by cplir-c**

### CODE REVIEW

**Comment by Shrimpstress** [comment]
Any chance you could change this to only occur in the spooky dimension and/or specific biomes?

File: src/main/resources/spookytime.mixins.json [hunk 1]
```diff
@@ -1,4 +1,15 @@
...
+    "required": true,
+    "package": "com.fabriccommunity.spookytime.mixin",
+    "compatibilityLevel": "JAVA_8",
+    "mixins": [
+        "LightmapTextureManagerMixin",
+	"AnvilBlockMixin",
+        "ServerWorldLightningMixin"
+    ],
+    "client": [
+=======
```
Comment (Shrimpstress) [inline src/main/resources/spookytime.mixins.json]: Please grab the latest `spookytime.mixins.json` and replace this one, and add your own mixin to it. Something happened and causes this monstrosity.

**Review by Shrimpstress — CHANGES_REQUESTED** [review]

**Comment by Shrimpstress** [comment]
This also completely breaks fluid rendering.

**Comment by Shrimpstress** [comment]
This has quite a lot of merge conflicts that need to be fixed.

**Review by vini2003 — CHANGES_REQUESTED** [review]
Quite a lot of merge conflicts need to be solved, as @Zundrel said.

**Comment by Shrimpstress** [comment]
I'm closing this for now due to inactivity. Feel free to make a new PR when you get around to continuing this, if you decide to.

**Pull request closed without merging**